MODULE.bazel

```diff
@@ -22,23 +22,10 @@ use_repo(
     "kotlinx_serialization_core_jvm",
     "kotlinx_serialization_json",
     "kotlinx_serialization_json_jvm",
-    "released_rules_kotlin",
 )
 
-# Once the released rules_koltin is defined, configure it.
-rules_kotlin_bootstrap_extensions = use_extension(
-    "//src/main/starlark/core/repositories:bzlmod_bootstrap.bzl",
-    "rules_kotlin_bootstrap_extensions",
-)
-use_repo(
-    rules_kotlin_bootstrap_extensions,
-    "released_com_github_google_ksp",
```

Comment:
Are these not being synced anymore for some reason?

Reply:
I removed the build-kotlin-with kotlin, so we don't use the released rules anymore.